17. Graph/Cheapest Flights Within K Stops.cpp

```diff
@@ -0,0 +1,52 @@
+#include <bits/stdc++.h>
```

Comment:
Please add question as comment too. Also, change file name, refer to readme.

Reply:
There are n cities connected by some number of flights. You are given an array flights where flights[i] = [fromi, to i, price i] indicates that there is a flight from city from i to city to i with cost price i.

You are also given three integers src, dst, and k, return the cheapest price from src to dst with at most k stops. If there is no such route, return -1.